server/controllers/articleController.js

```diff
@@ -23,10 +24,11 @@ export default class ArticleController {
     const { title, body, description } = req.body;
     const readTime = readingTime(body); // calculate the time it will take to read the article
     const imgUrl = (req.file ? req.file.secure_url : '');
+    const articleSlug = `${slug(title)}-${uuid()}`;
```

Comment:
I had to store the article slug in a variable so that I can use it on line  **31** and  **41**